terraform_files/baremetal_host/main.tf

```diff
@@ -68,8 +69,10 @@ resource "libvirt_domain" "host" {
     }
   }
 
+  machine = "q35"
```

Comment:
I think we should leave default behavior as-is and just allow templating this stuff if requested
You can never know what other environments are supporting and what's not, so even deciding to transition should be done gradually

Reply:
the same way we're templating ``vcpu``, ``memory`` and ``name`` in this file (to name a few). Those values are being set based on user input or other calculations

Reply:
For more info, please read: https://github.com/openshift/assisted-test-infra/blob/master/GUIDE.md#adding-a-new-e2e-flow

Reply:
okay, I see what you are saying now